.github/workflows/auto.yml

```diff
@@ -0,0 +1,44 @@
+name: Automatic CI for TFE Modules
+
+on:
+  pull_request:
+    branches:
+      - master
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  terraform_format:
+
+    name: Run terraform fmt
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Setup Terraform
+      uses: hashicorp/setup-terraform@v1
+      with:
+        terraform_version: 0.14.7
+
+    - name: Format Public Install
+      run: terraform fmt -check -recursive -- ${{ github.workspace }}
+
+  terraform_lint:
+
+    name: Run terraform-lint
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Setup Terraform Lint
+      uses: terraform-linters/setup-tflint@v1
+      with:
+        tflint_version: v0.26.0
+
+    - name: Lint Public Install
+      run: tflint --config ${{ github.workspace }}/.tflint.hcl .
```

Comment:
Should we be consistent in how we identify the working directory? We have `.` here but `${{ github.workspace }}` in the previous job. Additionally, I think we should include the submodules, examples, and tests here since we want to enforce linting for the entire repository. Unfortunately, tflint does not support recursive directories like `terraform fmt` so they need to be enumerated.

Reply:
I've taken care of the loops. There are now two additional steps to enumerate the modules/ and tests/ directories.